tests/default_missing_vals.rs

```diff
@@ -0,0 +1,116 @@
+#[cfg(test)]
+mod default_missing_vals {
+
+use clap::{App, Arg};
+
+#[test]
+fn opt_missing() {
+    let r = App::new("df")
+        .arg(Arg::new("color")
+            .long("color")
+            .value_name("WHEN")
+            .possible_values(&["always", "auto", "never"])
+            .multiple(true)
+            .overrides_with("color")
+            .about(
```

Comment:
Can we remove these `about`s? They are just adding noise here.

Reply:
Also, the `multiple` and `overrides_with` and all the other stuff. Please keep the tests small and to the point.

Reply:
Yeah, I agree.
I added this to the docs as an example and withdrew it from the tests.